signingscript/src/signingscript/task.py

```diff
@@ -56,6 +58,8 @@
         "autograph_rsa": sign_file_detached,
         "apple_notarization": apple_notarize,
         "apple_notarization_geckodriver": apple_notarize_geckodriver,
+        # This format is handled in script.py
+        # "apple_notarization_stacked": apple_notarize_stacked,
```

Comment:
We should be able to map this format to a function once the first todo above (about `sign`) is addressed. When you file that issue, please reference it here as well.